test/mtl/test_mtl_utils.py

```diff
@@ -0,0 +1,112 @@
+import unittest
```

Comment:
nit: rename `test_utils.py`

Reply:
I agree, but fun fact: tox won't let us have two tests with the same name:
```
import file mismatch:
imported module 'test_utils' has this __file__ attribute:
  /Users/braden/repos/snorkel/test/model/test_utils.py
which is not the same as the test file we want to collect:
  /Users/braden/repos/snorkel/test/mtl/test_utils.py
HINT: remove __pycache__ / .pyc files and/or use a unique basename for your test file modules
```

Reply:
It's actually a pytest issue:
https://github.com/pytest-dev/pytest/issues/529